responsibleai_text/responsibleai_text/rai_text_insights/rai_text_insights.py

```diff
@@ -264,16 +264,15 @@ def _validate_model(self, model: Any, test: pd.DataFrame,
         """
         if not isinstance(target_column, list):
             target_column = [target_column]
-        # Pick one row from test data
-        small_test_data = test.iloc[0:1].drop(
+        # Use full test dataset for validation
+        test_data = test.drop(
             target_column, axis=1)
-        small_test_data = get_text_columns(small_test_data, text_column)
-        small_test_data = small_test_data.iloc[0]
+        test_data = get_text_columns(test_data, text_column)
         if task_type != ModelTask.QUESTION_ANSWERING:
-            small_test_data = small_test_data.tolist()
+            test_data = test_data.tolist()
```

Comment:
it looks like tests are all failing with errors:

```
responsibleai_text/rai_text_insights/rai_text_insights.py:163: in __init__
    self._validate_rai_insights_input_parameters(
responsibleai_text/rai_text_insights/rai_text_insights.py:394: in _validate_rai_insights_input_parameters
    self._validate_model(model, test, target_column,
responsibleai_text/rai_text_insights/rai_text_insights.py:272: in _validate_model
    test_data = test_data.tolist()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self =                                                 text
0                            i didnt feel humiliated
1  i can go from feeling so hopeless to so damned...
2   im grabbing a minute to post i feel greedy wrong
name = 'tolist'

    def __getattr__(self, name: str):
        """
        After regular attribute access, try looking up the name
        This allows simpler access to columns for interactive use.
        """
        # Note: obj.x will always call obj.__getattribute__('x') prior to
        # calling obj.__getattr__('x').
        if (
            name not in self._internal_names_set
            and name not in self._metadata
            and name not in self._accessors
            and self._info_axis._can_hold_identifiers_and_holds_name(name)
        ):
            return self[name]
>       return object.__getattribute__(self, name)
E       AttributeError: 'DataFrame' object has no attribute 'tolist'
```

Reply:
changed back to raising user error in error analysis manager predict call